packages/app/src/runner/index.ts

```diff
@@ -229,15 +253,17 @@ function runSpecCT (spec: SpecFile) {
 }
 
 /**
- * Create a Spec IFrame. Used for loading the spec to execute in E2E
+ * Create an IFrame. If the Iframe is the spec iframe,
+ * this function is used for loading the spec to execute in E2E
  */
-function createSpecIFrame (specSrc: string) {
-  const el = document.createElement('iframe')
+function addIframe ({ $container, id, src, className }) {
```

Comment:
has slight mutation from [original](https://github.com/cypress-io/cypress/pull/18075/files#diff-bd8f567e7cefe6212c99b7760b613f12ffa518138f1d19aff5f3e5cf0c677bf9R195) as we may not be able to leverage `Cypress.$`